test/PoolManager.t.sol

```diff
@@ -989,18 +990,17 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
     }
 
     function test_collectProtocolFees_ERC20_accumulateFees_gas() public {
-        uint16 protocolFee = 1028; // 00000100 00000100
         uint256 expectedFees = 7;
 
-        feeController.setSwapFeeForPool(key.toId(), uint16(protocolFee));
+        feeController.setProtocolFeeForPool(key.toId(), MAX_FEE_BOTH_TOKENS);
         manager.setProtocolFee(key);
 
         (Pool.Slot0 memory slot0,,,) = manager.pools(key.toId());
-        assertEq(slot0.protocolFee, protocolFee);
+        assertEq(slot0.protocolFee, MAX_FEE_BOTH_TOKENS);
 
         swapRouter.swap(
             key,
-            IPoolManager.SwapParams(true, 10000, SQRT_RATIO_1_2),
+            IPoolManager.SwapParams(true, -10000, SQRT_RATIO_1_2),
```

Comment:
why is this sign flipped?

Reply:
since we're sending in exact input 1000 of token0 for token1